test/integration/host_js_integration_test.rb

```diff
@@ -0,0 +1,322 @@
+require 'integration_test_helper'
+require 'integration/shared/host_finders'
+
+class HostJSIntegrationTest < IntegrationTestWithJavascript
```

Comment:
Minor point on the file naming, though it matches the class name, the others don't use "integration" in the name as they're already in test/integration/ I guess.